modular_skyrat/modules/ghostcafe/code/ghostcafeturf.dm

```diff
@@ -0,0 +1,11 @@
+/turf/open/lava/planet
```

Comment:
```suggestion
/turf/open/lava/fake
```
Would be good to have this more descriptive for admin-finding purposes. Otherwise happy with this.